sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -85,6 +85,15 @@ toxTask "testPy38pandas-14", "py38-pandas-14"
 test.dependsOn "testPy38pandas-14"
 preCommitPy38.dependsOn "testPy38pandas-14"
 
+// Create a test task for each minor version of pytorch
+toxTask "testPy38pytorch-19", "py38-pytorch-19"
+test.dependsOn "testPy38pytorch-19"
+preCommitPy38.dependsOn "testPy38pytorch-19"
+
+toxTask "testPy38pytorch-110", "py38-pytorch-110"
+test.dependsOn "testPy38pytorch-110"
+preCommitPy38.dependsOn "testPy38pytorch-110"
```

Comment:
Does pytorch commonly make changes that will break us between minor versions? We test different minor versions for pandas because our special usage of pandas in the DataFrame API leads to breakages even between minor versions. In the case of pyarrow, every release is a major version, which is meant to communicate that the API can change (https://arrow.apache.org/docs/format/Versioning.html).

How will we keep this up to date as new version of pytorch come out?

Neither of these are blockers, but these are questions we should consider